spec/services/authn_context_resolver_spec.rb

```diff
@@ -291,42 +291,139 @@
       end
     end
 
-    context 'IAL2 service provider' do
-      it 'uses the IAL ACR if one is present' do
-        service_provider = build(:service_provider, ial: 2)
-
-        acr_values = [
-          'http://idmanagement.gov/ns/assurance/aal/1',
-          'http://idmanagement.gov/ns/assurance/ial/1',
-        ].join(' ')
-
-        result = AuthnContextResolver.new(
+    context 'with an IAL2 service provider' do
+      let(:service_provider) { build(:service_provider, ial: 2) }
+      subject do
+        AuthnContextResolver.new(
           user: user,
           service_provider: service_provider,
           vtr: nil,
           acr_values: acr_values,
-        ).resolve
-
-        expect(result.identity_proofing?).to eq(false)
-        expect(result.aal2?).to eq(false)
+        )
+      end
+      let(:result) { subject.resolve }
+
+      context 'if IAL ACR value is present' do
```

Comment:
Isn't the context really "when an IAL without identity proofing is requested"?

Reply:
the context is that the requested IAL ACR value takes precedence. but happy to change it to your wording if that's helpful